tools/interface/api.py

```diff
@@ -0,0 +1,24 @@
+import sys
```

Comment:
you can remove this file from the PR to keep it tidy (since this doesn't do anything useful right now)